domain-registrars/domain-syncing.md

```diff
@@ -10,6 +10,10 @@ The domain syncronisation functions allow you to propogate any domain expiry dat
 
 ## Domain Syncing
 
+Domains are only checked when they are in Active or Pendin Transfer status. A domain will never transfer to Cancelled status if a domain is subsequently deleted. 
```

Comment:
Thank you for your contribution. I think we may need to update this a little.

> A domain will never transfer to Cancelled status if a domain is subsequently deleted. 

This sentence is not necessarily true and relies entirely on the response from the respective modules `_Sync` method.

This method supports the return of either `active`, `cancelled`, `transferredaway` (all as booleans).
If ones module is able to determine that the domain was previously active with that registrar and was subsequently deleted, a `cancelled` response could be provided to the _Sync method.

Please do make any updates to this PR to reflect this as you see fit. If you would prefer our Documentation team provide extra clarity within this documentation, I would be happy to open a case internally with them.

Should I hear nothing back within 5 days, I will go ahead and open a case with our team to have this reviewed.

Thanks again for your contribution!